auth-examples/nodejs/server.js

```diff
@@ -0,0 +1,68 @@
+import express from 'express';
+import axios from 'axios';
+import crypto from 'crypto';
+import dotenv from 'dotenv';
+dotenv.config();
+
+const app = express();
+app.use(express.json());
+
+app.get('/test-auth', async (req, res) => {
+    try {
+        const apiKey = process.env.PUBLIC_KEY;
+        const privateKey = process.env.PRIVATE_KEY;
+
+        const termlyTimestamp = getTermlyTimestamp();
+        const authHeader = createAuthHeader(apiKey, privateKey, termlyTimestamp, 'GET', '', '/v1/authn', '');
+        console.log(`Authorization Header: ${authHeader}`);
+
+        const response = await axios.get('https://api.usw2.staging.trmly.net/v1/authn', {
```

Comment:
Shouldn't this be the prod URL? Or do we give them staging accounts?

Reply:
Right you are, slipped through in testing